modules/local/compute_gene_scores/environment.yml

```diff
@@ -0,0 +1,16 @@
+name: compute_gene_scores
+channels:
+  - conda-forge
+  - bioconda
+  - defaults
```

Comment:
defaults channel should be removed from the `environment.yml`.

See:
https://github.com/nf-core/modules/issues/5829